components/layout/grid/test/index.test.js

```diff
@@ -75,7 +75,7 @@ describe(json.name, () => {
       expect(container.innerHTML).to.not.have.lengthOf(0)
     })
 
-    it('should NOT extend classNames', () => {
+    it('should allows you to add custom classNames to add styles', () => {
```

Comment:
the test decription must only include what it test, not justify the purpose of its feature
```suggestion
    it('should allows you to add custom classNames', () => {
```